docs/mastering-plone/case.md

```diff
@@ -78,7 +78,7 @@ During the course of the training you will solve the following tasks.
 - Create a sponsor contenttype to manage sponsors
 - Store non-visible information about the sponsor in the sponsor-type
 - Display sponsors on all pages sorted by level
-- Allow some avilable to be configurable by a admin in a controlpanel
+- Allow some available to be configurable by a admin in a controlpanel
```

Comment:
This sentence does not make sense. It is missing a word after "available", or perhaps the intent is "availability"? I am not sure. Would you please look elsewhere in this Training to infer what it should be?

```suggestion
- Allow some available to be configurable by an admin in a control panel
```